### PR TITLE
Fix symbols for tasks

### LINKED ISSUE #3581: symbols packages being generated empty
We have turned off symbols package generation for now on this repo.

They were generating empty nupkgs and conflicting with the actual nupkgs when the sign tool was repackaging signing assemblies.

The fact that symbols are empty here are causing symbol checks to fail on VS insertions.

### PR DESCRIPTION
**Pull request by dsplaisted**
Create PDBs for tasks and put them in the directory where Arcade will upload them

Fixes #3581

### CODE REVIEW

**Comment by dsplaisted** [comment]
@livarcocc 

Private build is here: https://dev.azure.com/dnceng/internal/_build/results?buildId=329321&view=logs

Looking at the log I can see that it is uploading the PDBs in the PublishToAzureDevOpsArtifacts target.

**Comment by livarcocc** [comment]
@dsplaisted should the symbols package contain pdbs though? I cracked it open and all I see are dlls and targets.

**Comment by dsplaisted** [comment]
@livarcocc I did not think the package should contain pdbs, since we don't want to ship them in the SDK.  But maybe it needs to include the PDBs for them to flow through ngen, and it needs to be the core-sdk repo that makes sure they're not shipped in the SDK.

**Comment by livarcocc** [comment]
I don't understand. I mean the .symbols nupkg. Not the regular one. We shouldn't be consuming the .symbols nupkg, right?

**Comment by dsplaisted** [comment]
With this, we aren't producing symbol packages anymore.  We are putting the PDBs in a folder where they are picked up by the arcade symbol uploader.

**Comment by livarcocc** [comment]
I still saw a .symbols.nupkg artifact being produced through, which is what confused me.

**Comment by dsplaisted** [comment]
Where did you see the symbols nupkg being produced?

File: Directory.Build.props [hunk 1]
```diff
@@ -29,4 +30,14 @@
     <SdkLayoutDirectory>$(ArtifactsBinDir)$(Configuration)\dotnetLayout</SdkLayoutDirectory>
   </PropertyGroup>
 
+  <Target Name="OverrideSymbolsOutput" BeforeTargets="_CheckForInvalidConfigurationAndPlatform"
+          Condition="'$(DebugType)' != 'embedded'">
+
+    <ItemGroup>
+      <_DebugSymbolsOutputPath Remove="@(_DebugSymbolsOutputPath)" />
+      <_DebugSymbolsOutputPath Include="@(_DebugSymbolsIntermediatePath->'$(ArtifactsSymStoreDirectory)$(MSBuildProjectName)\$(TargetFramework.ToLowerInvariant())\%(Filename)%(Extension)')" />
+    </ItemGroup>
```
Comment (nguerrera) [inline Directory.Build.props]: Why do we need to do anything special. Doesn't arcade handle this automatically?

**Pull request merged**